stable/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.5.0
+version: 7.0.0
```

Comment:
If I'm not wrong, @carrodher suggested bumping the minor version, but you bumped the major version. I don't think we're introducing any incompatibility with these changes so I'd avoid bumping the major version:
```diff
- version: 7.0.0
+ version: 6.6.0
```

Reply:
Yes, sorry is my fault , i  bumped a minor.